pkg/cmd/infra/router/template.go

```diff
@@ -287,6 +294,77 @@ func parseCaptureHeaders(in string) ([]templateplugin.CaptureHTTPHeader, error)
 	return captureHeaders, nil
 }
 
+func sanitizeHeaderValue(headerValue string) string {
```

Comment:
Is it haproxy or something else that doesn't allow unescaped `'` in the header value?

Reply:
Why did you need to do it here as opposed to where you constructed the value in the ingress operator?

Reply:
Yes you need to perform escaping and quoting. Details are mentioned in the EP.

Reply:
If you already do it elsewhere, it shouldn't be needed here.